### PR TITLE
Rpm proof

### PR DESCRIPTION
**Pull request by Sammers21**
Related to #154

### CODE REVIEW

**Comment by Sammers21** [comment]
@g4s8, please review the Pull Request.

File: examples/rpm/run.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Post a wget package.
+curl -i  -X PUT  --data-binary "@wget-1.19.5-8.el8_1.1.x86_64.rpm" http://localhost:8080/my-rpm/wget-1.19.5-8.el8_1.1.x86_64.rpm
+
+# Start centos container and install wget from artipie.
+docker run --network artipie-rpm-demo --rm --name centos -it centos:8 \
+ /bin/bash -c "yum install -y yum-utils ; yum-config-manager --add-repo http://artipie:80/my-rpm/; yum --skip-broken --disablerepo='*' --enablerepo='80_my-rpm_' install wget"
```
Comment (olenagerasimova) [inline examples/rpm/run.sh:19]: @Sammers21 this line looks massive and difficult (time consuming) to read and understand. Maybe we could to split it into several steps?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@Sammers21 one suggestion, please, take a look

**Comment by Sammers21** [comment]
@olenagerasimova, could you review the Pull Request once again, please?

File: examples/rpm/run.sh [hunk 2]
```diff
@@ -16,7 +16,10 @@ curl -i  -X PUT  --data-binary "@wget-1.19.5-8.el8_1.1.x86_64.rpm" http://localh
 
 # Start centos container and install wget from artipie.
 docker run --network artipie-rpm-demo --rm --name centos -it centos:8 \
- /bin/bash -c "yum install -y yum-utils ; yum-config-manager --add-repo http://artipie:80/my-rpm/; yum --skip-broken --disablerepo='*' --enablerepo='80_my-rpm_' install wget"
+ /bin/bash -c "\
+  yum install -y yum-utils ; \
```
Comment (g4s8) [inline examples/rpm/run.sh:20]: @Sammers21 this test will not fail if any step fails, please use `/bin/bash -e -c "commands1; command2"` to fail CMD on step failure

**Review by g4s8 — CHANGES_REQUESTED** [review]
@Sammers21 please check my comment

**Comment by Sammers21** [comment]
@g4s8, review the PR one more time, please.

**Review by olenagerasimova — APPROVED** [review]
@Sammers21 thanks, looks good to me

**Review by olegmoz — APPROVED** [review]

**Comment by olegmoz** [comment]
@Sammers21 `ppd-lint` workflow fails for this PR. Please exclude binary files from pdd check and we are good to merge

**Review by olegmoz — CHANGES_REQUESTED** [review]
@Sammers21 pdd fix needed

**Comment by olegmoz** [comment]
@Sammers21 the proof is failing for some reason, could you please take a look?

**Comment by Sammers21** [comment]
@olegmoz, rpm proof is passing now.

**Comment by Sammers21** [comment]
@olegmoz, docker, and PyPI proof were broken in #457 PR and I've created an issue about it: #459

**Review by olegmoz — APPROVED** [review]
@Sammers21 thanks, merging it

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#418` is not assigned, can't get performer

**Pull request merged**